oamapps/postConfigure/helpers.cpp

```diff
@@ -48,7 +48,7 @@ string prompt;
 
 const char* pcommand = 0;
 
-extern bool noPrompting;
+bool noPrompting = false;
```

Comment:
quick fix for deletion of noPrompting global variable from postConfigure. in the future helpers.cpp should also undergo heavy cleanup.